packages/dev/core/src/Meshes/GaussianSplatting/gaussianSplattingMesh.ts

```diff
@@ -5,12 +5,12 @@ import { SubMesh } from "../subMesh";
 import type { AbstractMesh } from "../abstractMesh";
 import { Mesh } from "../mesh";
 import { VertexData } from "../mesh.vertexData";
-import { Tools } from "core/Misc/tools";
-import { Matrix, TmpVectors, Vector2, Vector3, Quaternion } from "core/Maths/math.vector";
+import { Matrix, TmpVectors, Vector2, Vector3 } from "core/Maths/math.vector";
 import { Logger } from "core/Misc/logger";
 import { GaussianSplattingMaterial } from "core/Materials/GaussianSplatting/gaussianSplattingMaterial";
 import { RawTexture } from "core/Materials/Textures/rawTexture";
 import { Constants } from "core/Engines/constants";
+import { appendSceneAsync } from "core/Loading/sceneLoader";

```

Comment:
This is now really strange to have here but I totally get the back compat :-)

Reply:
yes, having the load in this class was a mistake. :/